website/docs/d/private_link_service.html.markdown

```diff
@@ -0,0 +1,133 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_private_link_service"
+sidebar_current: "docs-azurerm-datasource-private-link-service"
+description: |-
+  Gets information about an existing Private Link Service
+---
+
+# Data Source: azurerm_private_link_service
+
+Use this data source to access information about an existing Private Link Service.
+
+
+## Private Link Service Usage
+
+```hcl
+data "azurerm_private_link_service" "example" {
+  resource_group_name = "acctestRG"
+  name                = "acctestpls"
+}
+
+output "private_link_service_id" {
+  value = "${data.azurerm_private_link_service.example.id}"
+}
+```
+
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the private link service.
+
+* `resource_group_name` - (Required) The name of the resource group.
+
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `location` - Resource location.
+
+* `alias` - The alias of the private link service.
+
+* `auto_approval` - One `auto_approval` block defined below.
+
+* `fqdns` - The list of Fqdn.
+
+* `ip_configurations` - One or more `ip_configuration` block defined below.
+
+* `load_balancer_frontend_ip_configurations` - One or more `load_balancer_frontend_ip_configuration` block defined below.
+
+* `network_interfaces` - One or more `network_interface` block defined below.
+
+* `private_endpoint_connections` - One or more `private_endpoint_connection` block defined below.
+
+* `type` - Resource type.
+
+* `visibility` - One `visibility` block defined below.
+
+* `tags` - Resource tags.
```

Comment:
this isn't in the schema?

Reply:
It is, but I didn't think it made sense so I removed it.